src/Visitor/NotMutableIgnoreVisitor.php

```diff
@@ -44,14 +45,25 @@
  */
 final class NotMutableIgnoreVisitor extends NodeVisitorAbstract
```

Comment:
I would say either `IgnoreNonMutableNodesVisitor` or `NonMutableNodesIgnorerVisitor` (I tend to prefer the later)